blockly-core/core/ui/block.js

```diff
@@ -558,6 +558,14 @@ Blockly.Block.prototype.getBox = function() {
 };
 
 /**
+ * Returns the padding of the SVG or null if none exists
+ * @return {Object} object with padding values for top, bottom, left, and right
+ */
+Blockly.Block.prototype.getSvgPadding = function() {
+  return this.svg_ && this.svg_.getPadding();
```

Comment:
I went back and forth on this for a while; my ultimate thinking was that SVG padding doesn't make sense at all if SVG is null, and that I wanted the structure of the `padding` object to be defined by BlockSvg.

You're right, though; I should totally have a null check below